task_05.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Week-08 warmup tasks"""
+
+import sys
+try:
+    x = int(raw_input("What is your blood pressure?"))
+except ValueError:
+    print "Invalid input"
+    sys.exit(0)
+
+if x > 0 and x <= 89:
```

Comment:
Ack! Instead of setting a variable _value_ you kept printing your result. Printing a result is a bit of a black hole -- it's visible to you, a human, but largely invisible to other programs which is why it's so important to store data in variables.